test/Dialect/FIRRTL/test.mlir

```diff
@@ -262,6 +262,18 @@ firrtl.module @BigIntTest(in %in: !firrtl.bigint, out %out: !firrtl.bigint) {
   %1 = firrtl.bigint -4
 }
 
+// CHECK-LABEL: ListTest
+// CHECK-SAME:  (in %in: !firrtl.list<string>, out %out: !firrtl.list<string>)
+firrtl.module @ListTest(in %in: !firrtl.list<string>, out %out: !firrtl.list<string>) {
+  firrtl.propassign %out, %in : !firrtl.list<string>
+}
+
+// CHECK-LABEL: MapTest
+// CHECK-SAME:  (in %in: !firrtl.map<bigint, string>, out %out: !firrtl.map<bigint, string>)
+firrtl.module @MapTest(in %in: !firrtl.map<bigint, string>, out %out: !firrtl.map<bigint, string>) {
+  firrtl.propassign %out, %in : !firrtl.map<bigint, string>
+}
+
```

Comment:
I see you have support for nested properties, which is great! Maybe add a test for that code path, like round-tripping `!firrtl.list<map<bigint, string>>`?

Reply:
Added test w/more complicated types, thanks!